transaction/core/tests/digest-test-vectors.rs

```diff
@@ -217,22 +217,22 @@ fn block_contents_digestible_test_vectors() {
     assert_eq!(
         results[0].1.hash().0,
         [
-            50, 42, 154, 188, 35, 247, 163, 87, 61, 59, 82, 164, 186, 4, 72, 210, 76, 213, 151,
-            223, 66, 55, 136, 186, 157, 33, 91, 190, 42, 17, 253, 118
+            141, 3, 218, 9, 245, 129, 12, 99, 192, 12, 107, 216, 4, 191, 254, 93, 247, 103, 110,
+            204, 77, 126, 154, 187, 198, 139, 58, 24, 191, 179, 59, 125
         ]
     );
     assert_eq!(
         results[1].1.hash().0,
         [
-            247, 31, 5, 1, 203, 138, 88, 158, 24, 207, 237, 227, 114, 101, 55, 137, 19, 180, 68,
-            177, 232, 192, 225, 22, 151, 232, 239, 137, 39, 230, 118, 219
+            254, 230, 153, 200, 129, 98, 221, 154, 120, 76, 165, 230, 183, 128, 60, 26, 202, 64,
+            171, 237, 164, 209, 9, 170, 109, 54, 133, 143, 110, 215, 199, 69
         ]
     );
     assert_eq!(
         results[2].1.hash().0,
         [
-            228, 130, 116, 249, 30, 143, 159, 34, 37, 156, 76, 25, 254, 136, 205, 157, 69, 167,
-            179, 156, 223, 149, 164, 110, 140, 105, 169, 233, 46, 235, 98, 194
+            215, 120, 30, 77, 126, 121, 230, 151, 225, 205, 247, 66, 101, 113, 221, 32, 122, 41,
+            29, 53, 98, 48, 72, 183, 219, 255, 228, 161, 47, 46, 245, 221
```

Comment:
these probably also should not be changing

Reply:
it might be okay because the origin block has no memos and those hashes are not changing

Reply:
nm, i didn't read far enough down in the file